docs/tutorials/introduction.ipynb

```diff
@@ -729,7 +729,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
Is this a new default, or something from your system?

Reply:
Don't know if it is a default but it appeared in other notebooks as well: 
```
docs/user-guide/coordinate-transformations.ipynb
434:   "display_name": "Python 3 (ipykernel)",

docs/tutorials/introduction.ipynb
732:   "display_name": "Python 3 (ipykernel)",

docs/tutorials/multi-d-datasets.ipynb
430:   "display_name": "Python 3 (ipykernel)",

tools/manual_testing/plot.ipynb
572:   "display_name": "Python 3 (ipykernel)"
```
Does this have any impact in the docs or when you open the notebooks?

Reply:
Let's see if it will cause trouble, CI passes so I'll assume it is fine.